.gitmodules

```diff
@@ -1,3 +1,3 @@
-[submodule "thronesdb-json-data"]
-	path = thronesdb-json-data
-	url = https://github.com/Alsciende/thronesdb-json-data.git
+[submodule "throneteki-json-data"]
+	path = throneteki-json-data
+	url = https://github.com/ystros/throneteki-json-data
```

Comment:
If we want to move the repo to a `throneteki` organization, we can. I'm not sure how much sense that makes unless 1. we're moving the main repo over there as well, 2. we anticipated more repos in the future - 2 repos seems kind of inconsequential for moving things around, 3. if we ever continue on generalizing the engine, it probably doesn't make sense for those repos to be under the name "throneteki".

Reply:
I'm happy keeping it wherever makes more sense, if there were to be a new organisation I'd move throneteki there as well, but it being where it is, is not a problem for me

Reply:
Since neither of us have strong opinions about it, it'll stay where it is for now. I'll make everyone collaborators on the repo.